valhalla/jawn/src/lib/handlers/LytixHandler.ts

```diff
@@ -0,0 +1,74 @@
+import { costOfPrompt } from "../../packages/cost";
+import {
+  HeliconeRequestResponseToPosthog,
+  PostHogEvent,
+  PosthogUserClient,
+} from "../clients/PosthogUserClient";
+import { PromiseGenericResult, err, ok } from "../shared/result";
+import { AbstractLogHandler } from "./AbstractLogHandler";
+import { HandlerContext } from "./HandlerContext";
+import crypto from "crypto";
+import * as Sentry from "@sentry/node";
+
+interface LytixModelIO {
+  apiKey: string;
+  modelName: string;
+  modelInput: string;
+  modelOutput: string;
+  metaData: Record<string, string>;
+  session: string;
+  userIdentifier: string;
+}
+
+export class LytixHandler extends AbstractLogHandler {
+  private lytixModelIO: LytixModelIO[] = [];
+  constructor() {
+    super();
+  }
+
+  public async handle(context: HandlerContext): PromiseGenericResult<string> {
+    if (!context.message.heliconeMeta.lytixKey) {
+      return await super.handle(context);
+    }
+
+    this.lytixModelIO.push({
+      apiKey: context.message.heliconeMeta.lytixKey,
+      modelName: context.processedLog.model ?? "",
+      modelInput: JSON.stringify(context.processedLog.request.body) ?? "",
+      modelOutput: JSON.stringify(context.processedLog.response.body) ?? "",
+      metaData: context.processedLog.request.properties ?? {},
+      session: "", // TODO when sessions are added
+      userIdentifier: context.message.log.request.userId,
+    });
+
+    return await super.handle(context);
+  }
+
+  public async handleResults(): PromiseGenericResult<string> {
+    for (const lytixModelIO of this.lytixModelIO) {
+      try {
+        // send lytix logs to lytix
+
+        await fetch("https://api.lytix.co/v1/metrics/modelIO", {
+          method: "POST",
+          headers: {
+            "lx-api-key": `Bearer ${lytixModelIO.apiKey}`,
+          },
+          body: JSON.stringify({
+            modelName: lytixModelIO.modelName,
+            modelInput: lytixModelIO.modelInput,
+            modelOutput: lytixModelIO.modelOutput,
+            metricMetadata: {},
```

Comment:
Just curious, was there a reason it wasn't?
```suggestion
            metricMetadata: lytixModelIO.metaData
```